### PR TITLE
Basic implementation for getCell with MissingCellPolicy.

### PR DESCRIPTION
**Pull request by waxxxd**
Will resolve the issues experienced in Issue #33 .

### CODE REVIEW

**Comment by monitorjbl** [comment]
Needs a unit test, but the code looks good. Add a test and I'll be good to merge this.

**Comment by waxxxd** [comment]
Will spin one up.

File: src/test/java/com/monitorjbl/xlsx/StreamingReaderTest.java [hunk 1]
```diff
@@ -579,4 +580,30 @@ public void shouldIgnoreSpreadsheetDrawingRows() throws Exception {
       }
     }
   }
+
+	@Test
+	public void testShouldReturnNullForMissingCellPolicy_RETURN_BLANK_AS_NULL() throws Exception {
```
Comment (monitorjbl) [inline src/test/java/com/monitorjbl/xlsx/StreamingReaderTest.java]: Looks good to me!

**Comment by waxxxd** [comment]
Literally pushed another line for the JUnit 10 seconds after the approval. New pull request?

**Comment by monitorjbl** [comment]
Sure, that's fine. What you had looked alright to me, but more tests are always good to have.

**Comment by waxxxd** [comment]
Is just a negative confirmation without the missing cell policy,  before the positive with the missing cell policy.

**Pull request merged**